src/components/Toolbar/Toolbar.jsx

```diff
@@ -37,9 +42,16 @@ function Toolbar() {
           <FaSearch tabIndex="0" />
         </li>
         <li>
-          <NavLink to="/sign-up" activeClassName="active-link">
-            Sign Up
-          </NavLink>
+        {user? (
```

Comment:
can you add the functionality in the HamburgerToolbar also?

Reply:
and would we be able to write tests to test the conditional view? I've never written tests that involve contexts?

Reply:
you can mock custom hooks by mocking the import file

Reply:
uhh the hamburger menu doesn't have a sign up option, which is why i didn't add the conditional. Should I add sign up as an option as well? Or only put a link for profile when there's a user?

Reply:
Ah I remember now that we don't have sign up or log in links for mobile cause they weren't in the mockups. We need to figure out how we want to handle that